### PR TITLE
MAINT-36266: Fix sending pasted image process and fix display of chat message composer on FireFox browser after pasting an image.

### PR DESCRIPTION
**Pull request by mdiha**
*Fix sending pasted image process form chat message composer:
Some images from browsers have a source url encoded to base64 format which contains characters as "+", and in cat message composer there is a shortcut for creating tasks by the "+" character so the pasted image is consider as adding a task and it throws an uncaught error, so i made sure to check whether it is an image or a simple text.
*Fix display of message composer when pasting an image on FireFox browser:
I changed the Vue js conditional rendering "v-if" to "v-show" for the element 'apps-container' since it has css transition property which crashes when adding an image in the content editable message composer of the chat.

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue [hunk 1]
```diff
@@ -205,8 +207,9 @@ export default {
       };
       let found = false;
       if(!this.miniChat) {
+        // shortcuts for specific applications actions
         this.composerApplications.forEach(application => {
-          if(application.shortcutMatches && application.shortcutMatches(newMessage)) {
+          if(application.shortcutMatches && application.shortcutMatches(newMessage) && !newMessage.includes('<img src=')) {
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue:212]: Could you just remove the image from the message before passing the message to the application shortcuts

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: application/webpack.dev.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const exoServerPath = "/exo-server/";
 
 let config = merge(webpackCommonConfig, {
   output: {
-    path: path.resolve(__dirname, exoServerPath + 'webapps/chat/')
+    path: path.resolve(__dirname, exoServerPath + '/webapps/chat/')
```
[the remark this answers is not in the document]
Comment (mdiha) [reply application/webpack.dev.js]: Okay will revert this change

File: application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue (hunk 1 above)
Comment (mdiha) [reply application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue]: Done

File: application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue [hunk 3]
```diff
@@ -203,8 +205,13 @@ export default {
         timestamp: Date.now(),
         user: eXo.chat.userSettings.username
       };
+      if (newMessage.includes('<img src=')) {
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue:208]: Could you use a regex for this like : 
`newMessage.replaceAll(/]*>/g,'');`

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue (hunk 3 above)
Comment (mdiha) [reply application/src/main/webapp/vue-app/components/ExoChatMessageComposer.vue:208]: Okay

**Review by ahamdi — APPROVED** [review]

**Pull request merged**